.github/workflows/update_iers.yml

```diff
@@ -0,0 +1,36 @@
+name: Auto-update IERS tables
+
+on:
+  schedule:
+    - cron: '0 0 * * 0'
+  workflow_dispatch:
+
+    jobs:
+  update-iers:
+    name: Auto-update IERS tables
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v2
+    - name: Download latest IERS files
+      run: ./update_builtin_iers.sh
+      working-directory: astropy/utils/iers/data
+    - name: Commit changes
```

Comment:
So, is this smart enough to not create a PR if there is no change?

Reply:
It is now, I made the ``git commit`` conditional, and the PR action is smart enough to see if there are no new commits.